app/utils/HouseConsignmentAnswersHelper.scala

```diff
@@ -0,0 +1,241 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils
+
+import cats.data.OptionT
+import cats.implicits._
+import models.{Index, UserAnswers}
+import pages._
+import pages.sections._
+import play.api.i18n.Messages
+import services.ReferenceDataService
+import uk.gov.hmrc.govukfrontend.views.viewmodels.content.Text
+import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
+import uk.gov.hmrc.http.HeaderCarrier
+import viewModels.sections.Section
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class HouseConsignmentAnswersHelper(userAnswers: UserAnswers, houseConsignmentIndex: Index, referenceDataService: ReferenceDataService)(implicit
+  messages: Messages,
+  hc: HeaderCarrier,
+  ec: ExecutionContext
+) extends AnswersHelper(userAnswers) {
+
+  def buildVehicleNationalityRow(transportMeansIndex: Index): Future[Option[SummaryListRow]] =
+    (for {
+      x <- OptionT.fromOption[Future](userAnswers.get(DepartureTransportMeansCountryPage(houseConsignmentIndex, transportMeansIndex)))
+      y <- OptionT.liftF(referenceDataService.getCountryNameByCode(x))
+    } yield transportRegisteredCountry(y)).value
+
+  def buildMeansOfTransportRows(idRow: Option[SummaryListRow], nationalityRow: Option[SummaryListRow]): Seq[SummaryListRow] =
```

Comment:
Can this be achieved like: 
idRow.map(Seq(_)).getOrElse(Seq.empty) ++ 
nationalityRow.map(Seq(_)).getOrElse(Seq.empty)